tools/report.collector

```diff
@@ -220,27 +225,35 @@ getconfig() {
 	fi
     done
 
-    crm_uuid -r > $target/$HB_UUID_F 2>&1
+    if which crm_uuid >/dev/null 2>&1; then
+        crm_uuid -r > $target/$HB_UUID_F 2>&1
+    fi
 
-    if
-	ps -ef | egrep -qs [c]rmd
-    then
-	crm_mon -1 2>&1 | grep -v '^Last upd' > $target/$CRM_MON_F
-	cibadmin -Ql 2>/dev/null > $target/${CIB_F}.live
+    if is_running crmd; then
+        dump_status_and_config
 	case $cluster in
 	    cman)             crm_node -p --cman > $target/$MEMBERSHIP_F 2>&1;;
 	    corosync|openais) crm_node -p --openais > $target/$MEMBERSHIP_F 2>&1;;
 	    heartbeat)        crm_node -p --heartbeat > $target/$MEMBERSHIP_F 2>&1;;
 	    *) 	              crm_node -p > $target/$MEMBERSHIP_F 2>&1;;
 	esac
 	echo "$host" > $target/RUNNING
+
+    elif is_running pacemaker_remoted; then
+        dump_status_and_config
+        echo "$host" > $target/RUNNING
+
     else
 	echo "$host" > $target/STOPPED
     fi
 
     if [ -f "$target/$CIB_F" ]; then
 	crm_verify -V -x $target/$CIB_F >$target/$CRM_VERIFY_F 2>&1
-	CIB_file=$target/$CIB_F crm configure show >$target/$CIB_TXT_F 2>&1
+        if which crm >/dev/null 2>&1 ; then
+            CIB_file=$target/$CIB_F crm configure show >$target/$CIB_TXT_F 2>&1
+        elif which pcs >/dev/null 2>&1 ; then
+            pcs config -f $target/$CIB_F >$target/$CIB_TXT_F 2>&1
```

Comment:
If only I had a dollar for every time I was looking at an archive and thought "we should really do pcs config here" and never came back :-)